.gitignore

```diff
@@ -1 +1,2 @@
-node_modules/
\ No newline at end of file
+node_modules/
+dist
```

Comment:
Good!

Please also introduce an `.h5pignore` file. It works the same like `.gitignore` files and tells the packer what file should not end up in the `.h5p`-file:

- Documentation
- src
- package-lock.json
- package.json 
- webpack.config.js

Cmp. https://snordian.de/2023/04/29/how-to-review-h5p-content-types/#h5pignore